packages/nodes-base/nodes/Set/Set.node.ts

```diff
@@ -24,6 +24,32 @@ export class Set implements INodeType {
 		inputs: ['main'],
 		outputs: ['main'],
 		properties: [
+			{
+				displayName: 'Display V1',
+				name: 'versionTest',
+				type: 'number',
+				displayOptions: {
+					show: {
+						'@version': [
+							1,
+						],
+					},
+				},
+				default: 1,
+			},
+			{
+				displayName: 'Display V2',
+				name: 'versionTest',
+				type: 'number',
+				displayOptions: {
+					show: {
+						'@version': [
+							2,
+						],
+					},
+				},
+				default: 2,
+			},
```

Comment:
I think these changes to the Set node can be undone, their purpose was to showcase how this can be used, right?